scripts/create_new_jsons.py

```diff
@@ -0,0 +1,271 @@
+"""create_new_jsons
+
+This script is used to create new json entries for new psu versions.
+This script will also update json indexes for the new psu versions.
+
+Note: YOU WILL STILL NEED TO:
+    1. Run sign.sh to sign the new psu version jsons
+    2. Ensure that the filenames for the `aqavit_results_link` enties are correct
+
+Requirements:
+    1. python 3.8 and higher
+
+Usage:
+  create_new_jsons --folder=<str> --new_version=<str> --psu_tag=<str>
```

Comment:
Nit: `python create_new_jsons.py ....`

Reply:
@joe-braley, I just tested this change out and it actually breaks the script. This is because: I am using the python library `docopt` to take in user inputs to the script. `docopt` needs a header in a specific format inorder to know how to function, and adding the word `python` here will break its expected format.

That said, if you look in the `README.md` file, you will find usage and example usage in the format you requested :)